package.json

```diff
@@ -18,9 +18,10 @@
   ],
   "scripts": {
     "eslint": "eslint bin tests",
-    "test": "npm run eslint && npm run test:coverage",
+    "test": "npm run eslint && npm run coverage",
```

Comment:
Doesn't this undo our script name normalization?

Reply:
reverted